apps/root/src/pages/transfer/components/recipient-address/components/contact-item/index.tsx

```diff
@@ -85,6 +90,30 @@ const ContactItem = ({ contact, onClickContact, onDeleteContact }: ContactItemPr
     );
   }, []);
 
+  const onClickContactHandler = () => {
+    if (contact.address === activeWallet?.address) {
+      snackbar.enqueueSnackbar(
```

Comment:
I thought it would be disadvantageous to allow the user to click on their own wallet as a recipient (having it as a contact). Doing so would close the modal, show the input error state, and then it would need to open the modal again. The validation in the input is always present for cases like pasting the address.
But for now, I've put this aside – we can take a closer look at this corner case later :)

Reply:
Cool, we should set this error on the input directly once they finish typing, but we can keep this for hardening